docs/docs/plugins/README.md

```diff
@@ -4,13 +4,11 @@
 - [Plugins List](/plugins/list.md)
 - [Plugins Technical Details](/plugins/details.md)
 
-:::note
+# Note
```

Comment:
The :::note is a special docusaurus thing ("admonition") to make it look nice 

![image](https://github.com/LAION-AI/Open-Assistant/assets/2178292/988524de-3b24-4dd3-ac8b-f01be760c0c4)

https://docusaurus.io/docs/markdown-features/admonitions

So i'd suggest leave them alone